File: python/001.py
# Name:     Multiples of 3 and 5
# Source:   https://projecteuler.net/problem=1


def sum_of_multiples(num_below, *nums_multiple):

    return sum(set(n for n in range(num_below) for num in nums_multiple if n % num == 0))


print(sum_of_multiples(1000, 3, 5))

File: javascript/001.js
// Name:     Multiples of 3 and 5
// Source:   https://projecteuler.net/problem=1


let sum_of_multiples = 0
for (let n = 1; n < 1000; n++) if (n % 3 === 0 || n % 5 === 0) sum_of_multiples += n

console.log(sum_of_multiples)
